pkg/asset/installconfig/basedomain.go

```diff
@@ -54,7 +54,9 @@ func (a *baseDomain) Generate(parents asset.Parents) error {
 		return platform.Azure.SetBaseDomain(zone.ID)
 	case gcp.Name:
 		var err error
-		a.BaseDomain, err = gcpconfig.GetBaseDomain(platform.GCP.ProjectID)
+		zone, err := gcpconfig.GetBaseDomain(platform.GCP.ProjectID)
+		a.BaseDomain = zone.DNSName
+		platform.GCP.PublicZoneID = zone.ID
```

Comment:
this will panic is the zone is nil.. like in cases there is an error.

Reply:
Fixed.

Reply:
@jhixson74 how is this fixed, i still see we are using `zone` when err could be not-nil, can you provide a reasoning as to why you think this is fixed?